app/controllers/coronavirus_form/data_export_controller.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+class CoronavirusForm::DataExportController < ApplicationController
+  def show
+    render plain: produce_csv(usage_statistics)
+  end
+
+private
+
+  def usage_statistics
+    questions = I18n.t("coronavirus_form.groups").map { |_, v| v[:questions] }.reduce(:merge)
+
+    start_date = Date.parse(sanitize(params[:start_date])).beginning_of_day
+    end_date = Date.parse(sanitize(params[:end_date])).end_of_day
+
+    results = {}
+    questions.each_key do |question|
+      question_text = questions.dig(question, :title)
+      counts = FormResponse.
+        where(created_at: start_date..end_date).
+        select(%{created_at::date, form_response -> '#{question}', COUNT(*)}).
+        group(%{created_at::date, form_response -> '#{question}'}).
+        pluck(%{created_at::date, form_response -> '#{question}', COUNT(*)}).
+        reject { |v| v[1].nil? }.
```

Comment:
If v is the group, what's v[1]?

Reply:
It's the second column returned by the select statement, i.e. the response to the question.  I'm removing `nil` since the response includes `nil` where a user never answered a question.